data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -115,6 +117,12 @@ private void initialize(
       app.config.server(
           () -> {
             jettyServer = new Server(configuration.getRestApiPort());
+            LOG.debug("Setting Max URL length to {}", configuration.getMaxUrlLength());
+            for (Connector c : jettyServer.getConnectors()) {
+              c.getConnectionFactory(HttpConnectionFactory.class)
+                  .getHttpConfiguration()
+                  .setRequestHeaderSize(configuration.getMaxUrlLength());
+            }
```

Comment:
Is it guaranteed that there is always a `HttpConnectionFactory` class to get or should we check for a `null` here?